assets/email/failed-login-attempt.txt

```diff
@@ -0,0 +1,9 @@
+Hi there!
+
+This is just an email to warn you that there was a failed attempt to login to your account in Padloc, from {{ location }}.
+
+If this was you, there's no action necessary, otherwise you might want to make sure your trusted devices haven't been compromised, or to remove potentially compromised devices from your trusted list inside the app.
+
+Note that if this happened on a trusted device, it was already removed from the trusted devices list, automatically.
+
+This email was sent to you by Padloc (https://padloc.app). If you have any questions, please don't hesitate to contact us at support@padloc.app!
```

Comment:
Just a minor thing, but can we add a

> Have a great day!
> 
> Your Padloc Team

Reply:
I intentionally removed that because this didn't seem like a "feel good" email, but I can add it back.

Reply:
True. Just "Your Padloc Team" would be fine too I guess.